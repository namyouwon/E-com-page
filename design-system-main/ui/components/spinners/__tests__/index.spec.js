/* eslint-env jest */
import React from 'react';

import Spinner from '../index.react';

import createHelpers from '../../../../jest.helpers';

const { matchesMarkup } = createHelpers(__dirname);

it('renders spinner', () =>
  matchesMarkup(<Spinner className="slds-spinner_medium" />));

it('renders delayed spinner', () =>
  matchesMarkup(
    <Spinner className="slds-spinner_delayed slds-spinner_medium" />
  ));
