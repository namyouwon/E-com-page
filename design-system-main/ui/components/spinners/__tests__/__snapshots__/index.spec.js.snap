// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders delayed spinner 1`] = `
<div
  className="slds-spinner_container"
>
  <div
    className="slds-spinner slds-spinner_delayed slds-spinner_medium"
    role="status"
  >
    <span
      className="slds-assistive-text"
    >
      Loading
    </span>
    <div
      className="slds-spinner__dot-a"
    />
    <div
      className="slds-spinner__dot-b"
    />
  </div>
</div>
`;

exports[`renders spinner 1`] = `
<div
  className="slds-spinner_container"
>
  <div
    className="slds-spinner slds-spinner_medium"
    role="status"
  >
    <span
      className="slds-assistive-text"
    >
      Loading
    </span>
    <div
      className="slds-spinner__dot-a"
    />
    <div
      className="slds-spinner__dot-b"
    />
  </div>
</div>
`;
